scripts/dropins.js

```diff
@@ -0,0 +1,33 @@
+// Drop-in Tools
+import { events } from '@dropins/elsie/event-bus.js';
+import * as mesh from '@dropins/elsie/fetch-graphql.js';
+import { initializers } from '@dropins/elsie/initializer.js';
+
+// Drop-ins
+import * as cart from '@dropins/storefront-cart/api.js';
+import * as product from '@dropins/storefront-pdp/api.js';
+import * as checkout from '@dropins/storefront-checkout/api.js';
+import * as orderConfirmation from '@dropins/storefront-order-confirmation/api.js';
+
+// Libs
+import { getConfigValue } from './configs.js';
+
+export default async function initializeDropins() {
+  // Set Commerce Mesh endpoint
+  mesh.setEndpoint(await getConfigValue('commerce-core-endpoint'));
```

Comment:
Calling the import `mesh` can be confusing - as it's unrelated to actual API Mesh.

Reply:
Fixed. Added a space for global initialization like Cart and Auth and blocks initialize their dropins